packages/rest-typings/src/v1/omnichannel.ts

```diff
@@ -2821,11 +2821,13 @@ export type GETLivechatRoomsParams = PaginatedRequest<{
 	closedAt?: string;
 	agents?: string[];
 	roomName?: string;
-	departmentId?: string;
+	// TODO: support only arrays on next major
+	departmentId?: string | string[];
```

Comment:
maybe make a separate `departments` param that accepts an array, while keeping `departmentId` as a string and flagged as deprecated?

Reply:
This is to reduce UI changes. UI will keep sending either a string or an array without knowing there's anything happening.